blockchain/config.go

```diff
@@ -89,7 +91,11 @@ var (
 		BloomfilterIndexDBPath:     "/var/data/bloomfilter.index.db",
 		CandidateIndexDBPath:       "/var/data/candidate.index.db",
 		StakingIndexDBPath:         "/var/data/staking.index.db",
+		SGDIndexDBPath:             "/var/data/sgd.index.db",
 		ContractStakingIndexDBPath: "/var/data/contractstaking.index.db",
+		BlobStoreDBPath:            "/var/data/blob.db",
+		BlobStoreRetentionDays:     20,
+		BlobPurgeInterval:          time.Hour,
```

Comment:
enforce 20 days of blob storage

Reply:
After reviewing the implementation of blobstore, we may need to make the following two changes:
1. Change `BlobStoreRetentionDays` to `BlobStoreRetentionBlocks`
2. instead of deleting blobs every hour in a batch, deleting them when `PutBlock` is called. Because deleting in a batch may delay `PutBlock` and cause possible delay to the whole network.

Reply:
recommend extra 1 day for enhancing robustness of blobsync, like [bsc](https://github.com/bnb-chain/bsc/blob/master/params/protocol_params.go#L195C2-L195C36) does

Reply:
1. about changing days to blocks, there is one consideration. If there's a possibility our block time will further reduce (like to 3 seconds), using blocks will change/reduce the storage retention time. Using days is more straightforward measurement of storage
2. about deleting blobs, the thing is `PutBlock` may not be called in very long time (like hours or even days), in such case there's a larger batch and bigger chance of delay. And in the current implementation, deleting blobs are done by a separate go-routine, and there's no lock contention with the `PutBlock()` -- only a light `atomic.LoadUint64()`, so it won't interfere or cause delay to the whole network.